playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -51,8 +51,19 @@
   tags:
     - install
 
-- name: Ensure kubectl is installed
```

Comment:
out of curiosity -- does it make sense that we remove all these deployer package and client install tasks from deploy_osh and include the airship-setup-deployer role in the osh deployment playbook?

Reply:
Of course, that's why I wrote that there's a conflict with https://github.com/SUSE-Cloud/socok8s/pull/265. 

The end result is to have only deployment tasks at one place only